src/mlpack/core/boost_backport/boost_backport_serialization.hpp

```diff
@@ -25,6 +25,14 @@
   #include <boost/serialization/unordered_map.hpp>
 #endif
 
+#if BOOST_VERSION < 106100
```

Comment:
I suggest to create a file e.g. `boost_backport_string_view.hpp` since this file is created for the serialization only.

Reply:
Created a new file.